import_email_events.py

```diff
@@ -0,0 +1,17 @@
+#
+# Script to import "email event" metrics from S3 into redshift.
+#
+
+import import_events
+SCHEMA = """
+    type VARCHAR(30) NOT NULL,
+    locale VARCHAR(40),
+    domain VARCHAR(40),
+    template VARCHAR(64),
+    flow_id VARCHAR(64)
```

Comment:
IIUC some of these events will not have a flow_id, e.g. emails that we send related to "change password" which is not part of a flow.